Documentation/coding-guidelines/clr-code-guide.md

```diff
@@ -8,7 +8,85 @@ Written in 2006, by:
 - Mike Stall ([@mikestall](https://github.com/mikestall))
 - Rudi Martin ([@Rudi-Martin](https://github.com/Rudi-Martin))
 
-# Why you must read this document
+# Contents
+
+* [1 Why you must read this document](#1)
+  * [1.1 Rules of the Code](#1.1)
+  * [1.2 How do I &lt;insert common task&gt;?](#1.2)
+* [2 Rules of the Code (Unmanaged)](#2)
```

Comment:
:+1: 

For markdown, probably instead of:

```markdown
* [1 Why you must read this document](#1)
  * [1.1 Rules of the Code](#1.1)
  * [1.2 How do I &lt;insert common task&gt;?](#1.2)
* [2 Rules of the Code (Unmanaged)](#2)
```

We can use:

```markdown
1. [Why you must read this document](#1)
  1.1. [Rules of the Code](#1.1)
  1.2. [How do I &lt;insert common task&gt;?](#1.2)
2. [Rules of the Code (Unmanaged)](#2)
```

(notice the period after number)

which will result in:

1. [Why you must read this document](#1)
  1. [Rules of the Code](#1.1)
  2. [How do I &lt;insert common task&gt;?](#1.2)
2. [Rules of the Code (Unmanaged)](#2)